File: MultiBoxJavaCore/src/sk/hackcraft/multibox/net/data/GetPlayerStateResultData.java
package sk.hackcraft.multibox.net.data;

import sk.hackcraft.multibox.model.Multimedia;

public class GetPlayerStateResultData
{
	private final Multimedia multimedia;
	private final int playbackPosition;
	private final boolean playing;
	
	public GetPlayerStateResultData(Multimedia multimedia, int playbackPosition, boolean playing)
	{
		this.multimedia = multimedia;
		this.playbackPosition = playbackPosition;
		this.playing = playing;
	}
	
	public Multimedia getMultimedia()
	{
		return multimedia;
	}

	public int getPlaybackPosition()
	{
		return playbackPosition;
	}

	public boolean isPlaying()
	{
		return playing;
	}
}

File: MultiboxClientCSharpCore/Net/Data/GetPlayerStateResultData.cs
using MultiboxClientCSharpCore.Model;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace MultiboxClientCSharpCore.Net.Data
{
	public class GetPlayerStateResultData
	{
		public GetPlayerStateResultData(MultimediaItem multimedia, int playbackPosition, bool playing)
		{
			this.Multimedia = multimedia;
			this.PlaybackPosition = playbackPosition;
			this.Playing = playing;
		}
	
		public MultimediaItem Multimedia
		{
			get;
			private set;
		}

		public int PlaybackPosition
		{
			get;
			private set;
		}

		public bool Playing
		{
			get;
			private set;
		}
	}
}
